client/src/glossary/glossary.js

```diff
@@ -185,7 +170,23 @@ export default class Glossary extends React.Component {
           <TM k="glossary" />
         </h1>
         <ScrollToTargetContainer target_id={active_key}>
-          {!is_a11y_mode && <BackToTop />}
+          {!is_a11y_mode && (
+            <FloatingButton
+              button_text={text_maker("back_to_top")}
+              aria_label={text_maker("back_to_top")}
+              showWithScroll={true}
+              handleClick={() => scroll_to_top()}
+              tabIndex={-1}
```

Comment:
This was just mimicking the original setup for the BackToTop component. I believe the idea is that the button isn't keyboard focusable because of its placement in the DOM and every glossary definition has a hidden link back to the top when navigating via keyboard.